### DISCUSSION TITLE
Where in the codebase is the right-click context menu that allows you to do things such as change case, and insert emojis?

### DISCUSSION THREAD

**Discussion by crisxh** [jendrikseipp/rednotebook#695] [Q&A] [answered]
I was going through the repo but I can't find anything about the context menu in the code, and the word emoji doesn't come up at all when I try to search through the files with vscode. I am kind of a noob when it comes to programming so apologies if I am missing something obvious here. Thanks.

**Comment by jendrikseipp** [comment, marked as answer]
I think this context menu is generated by the underlying GTK library.

**Reply by crisxh** [reply]
Ok thanks. I will look into that.
